build/common.ps1

```diff
@@ -761,53 +761,83 @@ Function Build-ClientsPackages {
         [int]$BuildNumber = (Get-BuildNumber),
         [ValidateSet(14,15)]
         [int]$ToolsetVersion = $DefaultMSBuildVersion,
-        [string]$KeyFile
+        [string]$KeyFile,
+        [switch]$SkipILMerge
     )
 
-    $exeProjectDir = [io.path]::combine($NuGetClientRoot, "src", "NuGet.Clients", "NuGet.CommandLine")
-    $exeProject = Join-Path $exeProjectDir "NuGet.CommandLine.csproj"
-    $exeNuspec = Join-Path $exeProjectDir "NuGet.CommandLine.nuspec"
-    $exeInputDir = [io.path]::combine($Artifacts, "NuGet.CommandLine", "${ToolsetVersion}.0", $Configuration)
-    $exeOutputDir = Join-Path $Artifacts "VS${ToolsetVersion}"
+    if (-not $SkipILMerge){
+        $exeProjectDir = [io.path]::combine($NuGetClientRoot, "src", "NuGet.Clients", "NuGet.CommandLine")
+        $exeProject = Join-Path $exeProjectDir "NuGet.CommandLine.csproj"
+        $exeNuspec = Join-Path $exeProjectDir "NuGet.CommandLine.nuspec"
+        $exeInputDir = [io.path]::combine($Artifacts, "NuGet.CommandLine", "${ToolsetVersion}.0", $Configuration)
+        $exeOutputDir = Join-Path $Artifacts "VS${ToolsetVersion}"
+
+        # Build and pack the NuGet.CommandLine project with the build number and release label.
+        Build-ClientsProjectHelper `
+            -SolutionOrProject $exeProject `
+            -Configuration $Configuration `
+            -ReleaseLabel $ReleaseLabel `
+            -BuildNumber $BuildNumber `
+            -ToolsetVersion $ToolsetVersion `
+            -Rebuild
+
+        Invoke-ILMerge `
+            -InputDir $exeInputDir `
+            -OutputDir $exeOutputDir `
+            -KeyFile $KeyFile
+
+        $opts = 'pack', $exeNuspec
+        $opts += '-BasePath', $exeOutputDir
+        $opts += '-OutputDirectory', $Nupkgs
+        $opts += '-Symbols'
+        $opts += '-Version', "$PackageReleaseVersion-$ReleaseLabel-$BuildNumber"
+        & $NuGetExe $opts
+
+        # Build and pack the NuGet.CommandLine project with just the release label.
+        Build-ClientsProjectHelper `
+            -SolutionOrProject $exeProject `
+            -Configuration $Configuration `
+            -ReleaseLabel $ReleaseLabel `
+            -BuildNumber $BuildNumber `
+            -ToolsetVersion $ToolsetVersion `
+            -ExcludeBuildNumber `
+            -Rebuild
+
+        Invoke-ILMerge `
+            -InputDir $exeInputDir `
+            -OutputDir $exeOutputDir `
+            -KeyFile $KeyFile
+
+        $opts = 'pack', $exeNuspec
+        $opts += '-BasePath', $exeOutputDir
+        $opts += '-OutputDirectory', $ReleaseNupkgs
+        $opts += '-Symbols'
+        if ($ReleaseLabel -Ne 'rtm') {
```

Comment:
We should only have to have this logic once. Maybe have a `$releaseNupkgVersion` shared between the two packages.